### PR TITLE
Create debug-statefulset.md

### PR DESCRIPTION
**Pull request by ihuibin**
Signed-off-by: bin.hui <bin.hui@daocloud.io>

### CODE REVIEW

File: content/zh/docs/tasks/debug/debug-application/debug-statefulset.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+reviewers:
+- bprashanth
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
+title: Debug a StatefulSet
```
Comment (sftim) [inline content/zh/docs/tasks/debug/debug-application/debug-statefulset.md:10]: This title isn't localized, and the reviewer list looks wrong.

File: content/zh/docs/tasks/debug/debug-application/debug-statefulset.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+## 调试一个 StatefulSet
+
+为了列出属于StatefulSet的所有pod，它们有一个标签' app=myapp '，
+你可以使用以下方法:
+
+```shell
+kubectl get pods -l app=myapp
+```
+
+如果你发现任何pod列表上的状态长时间处于“Unknown”或“Terminating”状态，参见[Deleting StatefulSet Pods](/docs/tasks/run-application/delete- StatefulSet /)查看如何处理它们的说明。
```
Comment (sftim) [inline content/zh/docs/tasks/debug/debug-application/debug-statefulset.md:34]: This link looks wrong.

**Review by sftim — CHANGES_REQUESTED** [review]
I don't think this page has been localized with enough care.

**Comment by tengqm** [comment]
@ihuibin Please refer to https://kubernetes.io/zh/docs/contribute/localization_zh/ for the style guide.

**Comment by Sea-n** [comment]
It's duplicated with another PR, next time you can assign yourself to [umbrella issues](https://github.com/kubernetes/website/issues?q=is%3Aissue+zh+Umbrella) to avoid conflict.
- #33372

**Comment by tengqm** [comment]
/close
closing as a dup.

**Pull request closed without merging**